tests/integ/test_horovod.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import tarfile
+import tempfile
+
+import boto3
+import pytest
+
+from sagemaker.tensorflow import TensorFlow
+from six.moves.urllib.parse import urlparse
+import tests.integ as integ
+from tests.integ import timeout, vpc_test_utils
+
+horovod_dir = os.path.join(os.path.dirname(__file__), '..', 'data', 'horovod')
+
+
+@pytest.mark.parametrize('instance_type', ['ml.p3.2xlarge', 'ml.p2.xlarge'])
+def test_horovod_tf_benchmarks(sagemaker_session, instance_type):
```

Comment:
discussed online - i believe this test should live in tf-container, too heavy for sdk testing purposes.